static/app/utils/theme/theme.chonk.tsx

```diff
@@ -0,0 +1,185 @@
+const _chonkShared = {
+  space: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
+    subMicroscoptic: 1,
+    microscoptic: 2,
+    mini: 4,
+    small: 6,
+    medium: 8,
+    large: 12,
+    huge: 16,
+  },
+  borderRadius: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
+    subMicroscoptic: 1,
+    microscoptic: 2,
+    mini: 3,
+    small: 4,
+    medium: 5,
+    large: 6,
+  },
+};
+
+const _chonkLightColors = {
+  // @TODO(jonasbadalic): add explanation about static and dynamic color differences and intended usage
+  static: {
+    black: '#181225',
+    white: '#FFFFFF',
+
+    blurple100: '#463299',
+    blurple200: '#5E42CC',
+    blurple300: '#694BE5',
+    blurple400: '#7553FF',
+
+    green400: '#00F261',
+    green300: '#00E55C',
+    green200: '#7553FF',
+    green100: '#00D957',
+
+    gold400: '#FFD00E',
+    gold300: '#FDB81B',
+    gold200: '#E5BB0D',
+    gold100: '#E07518',
+
+    red400: '#E50045',
+    red300: '#E565A9',
+    red200: '#CC5A97',
+    red100: '#B24E84',
+
+    pink400: '#FF70BC',
+    // @TODO(jonasbadalic): duplicates?
```

Comment:
Yep, so I don't have the Pink colour scale done yet.

In the meantime, I'd suggest we just use what we have today for both light and dark.

I'll prioritise this task after getting the form elements out the door

Reply:
@Jesse-Box this doesnt have to be final to merge. We can live with it being like this and you can just make a PR to add the pink scale when you are ready :)